### PR TITLE
M3-2797 Refactor/Improve StackScripts Table styles and mobile handling

### PR DESCRIPTION
**Pull request by alioso**
## Refactor/Improve StackScripts Table styles and mobile handling

The StackScripts landing page was in great need of work - the work was help up waiting for pagination. Since it is not happening anytime soon we still need to update the styles and give users a better experience.

The stackScripts table on the create workflow was also updated and since we have a button there for details I removed some of the table cells so that's it's easier to read and provides a better experience.

**To be tested on various breakpoints and browsers.**

## Type of Change
- Non breaking change ('update')

### CODE REVIEW

File: src/components/StackScript/StackScript.tsx [hunk 1]
```diff
@@ -107,32 +131,47 @@ export class StackScript extends React.Component<CombinedProps> {
...
+            <strong>{deployments_total}</strong> deployments
+          </Typography>
+          <Typography className={classes.deploymentSection}>
+            <strong>{deployments_active}</strong> still active
+          </Typography>
+          <Typography className={classes.deploymentSection}>
+            <strong>Last revision: </strong>
+            <DateTimeDisplay
+              value={updated}
+              humanizeCutoff={'never'}
```
Comment (martinmckenna) [inline src/components/StackScript/StackScript.tsx]: I think we should cutoff by month. Anything after 30 days, I'd like to see the exact timestamp for.

File: src/components/StackScript/StackScript.tsx [hunk 2]
```diff
@@ -107,32 +131,47 @@ export class StackScript extends React.Component<CombinedProps> {
...
+          >
+            <strong>Compatible with:</strong>
+            <span className={classes.compatibleImages}>{compatibleImages}</span>
           </Typography>
+          <Divider className={classes.divider} />
         </div>
-        <Typography variant="h3" className={classes.scriptHeading}>
-          Script:
+        <Typography className={classes.scriptHeading}>
+          <strong>Script:s</strong>
```
Comment (martinmckenna) [inline src/components/StackScript/StackScript.tsx]: stray `s` here

File: src/features/StackScripts/SelectStackScriptPanel/StackScriptSelectionRow.tsx [hunk 3]
```diff
@@ -113,7 +126,7 @@ export class StackScriptSelectionRow extends React.Component<
           >
             {renderLabel()}
           </TableCell>
-          <TableCell>
+          {/* <TableCell>
```
Comment (johnwcallahan) [inline src/features/StackScripts/SelectStackScriptPanel/StackScriptSelectionRow.tsx]: If we need to leave this commented-out code here, could we leave a small comment explaining what it is and why it's still here?

File: src/features/StackScripts/StackScriptBase/StackScriptBase.tsx [hunk 4]
```diff
@@ -459,10 +459,12 @@ const withStackScriptBase = (isSelecting: boolean) => (
                 />
               </div>
               <Table
-                isResponsive={false}
+                isResponsive={!isSelecting && true}
```
Comment (johnwcallahan) [inline src/features/StackScripts/StackScriptBase/StackScriptBase.tsx]: I don't _think_ `&& true` is doing anything here.

**Pull request merged**